developer/tutorials/articles/03-upgrading-code-to-liferay-7.2/01-intro.markdown

```diff
@@ -0,0 +1,233 @@
+---
+header-id: upgrading-code-to-product-ver
+---
+
+# Upgrading Code to @product-ver@
+
+[TOC levels=1-4]
+
+Upgrading to @product-ver@ involves migrating your installation and code (your
+custom apps) to the new version. You'll learn how to upgrade your code in this
+section.
+
+These tutorials assume you're using the
+[Code Upgrade Planner](/docs/7-2/reference/-/knowledge_base/r/code-upgrade-planner).
```

Comment:
@codyhoag and @gamerson , @jtydhr88 and I think we can use "Liferay Upgrade Planner" instead of "Code Upgrade Planner", please help update.

Reply:
I think so, as the planner do not only support code upgrade also support DB or something else

Reply:
Just changed the docs to reference the tool as *Liferay Upgrade Planner*.